### PR TITLE
cedar wasm example

### PR DESCRIPTION
**Pull request by Swolebrain**
*Issue #, if available:*

n/a

*Description of changes:*

adds a wasm example

### CODE REVIEW

File: cedar-wasm-example/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.ts",
+  "scripts": {
+    "test": "jest",
+    "start": "ts-node main.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
```
Comment (khieta) [inline cedar-wasm-example/package.json:12]: I think for our other code we generally use Apache-2.0. Is there a reason to use MIT here?

**Review by khieta — COMMENTED** [review]
So cool! Thanks for the contribution! I think this will be really useful for consumers of the Wasm code. And once we get CI set up, it will help our team avoid unintentionally breaking the Wasm bindings 🙂  

Could you add a README with instructions on how to build & run tests?

**Review by andrewmwells-amazon — APPROVED** [review]

**Comment by andrewmwells-amazon** [comment]
Build failure is unrelated.

**Review by shaobo-he-aws — APPROVED** [review]

**Comment by andrewmwells-amazon** [comment]
Downstram failure is unrelated (fixed in https://github.com/cedar-policy/cedar/pull/1015)

**Pull request merged**